cli/package.json

```diff
@@ -59,7 +59,9 @@
   },
   "dependencies": {
     "fast-glob": "^3.3.2",
-    "lodash-es": "^4.17.21"
+    "install": "^0.13.0",
```

Comment:
You still have an extra dependency here that has been added erroneously it seems

Reply:
Awesome thank you for pointing it out. I removed it and re-ran npm install. hopefully its all fixed now 😁